nvflare/fuel/hci/zip_utils.py

```diff
@@ -39,12 +40,29 @@ def _path_join(base: str, *parts: str) -> str:
     return path.replace("\\", "/")
 
 
-def remove_leading_dotdot(path):
+def remove_leading_dotdot(path: str) -> str:
     while path.startswith("../"):
         path = path[3:]
     return path
 
 
+def split_path(path: str) -> (str, str):
+    """Split a path into prefix and folder name
+
+    Args:
+        path: Path to split
+
+    Returns: A tuple of (prefix, folder_name)
+    """
+
+    if path.endswith("/"):
```

Comment:
Same here, but can be in a follow up improvement PR since we have other places using this "/"

Reply:
In ZIP file, path is always separated by slashes, no back slashes.